src/coreclr/tools/superpmi/superpmi/superpmi.cpp

```diff
@@ -380,334 +372,367 @@ int __cdecl main(int argc, char* argv[])
             continue;
         }
 
-        if (jit == nullptr)
-        {
-            SimpleTimer stInitJit;
+        // Save the stored CompileResult
+        mc->originalCR = mc->cr;
 
-            jit = JitInstance::InitJit(o.nameOfJit, o.breakOnAssert, &stInitJit, mc, o.forceJitOptions, o.jitOptions);
+        // Compile this method context as many times as we have been asked to (default is once).
+        for (int iter = 0; iter < o.repeatCount; iter++)
+        {
             if (jit == nullptr)
             {
-                // InitJit already printed a failure message
-                return (int)SpmiResult::JitFailedToInit;
-            }
+                SimpleTimer stInitJit;
```

Comment:
Note that most of the changes starting here are because everything in the `iter` loop was indented 4 spaces.